client/cody/webviews/UserHistory.tsx

```diff
@@ -75,27 +75,32 @@ export const UserHistory: React.FunctionComponent<React.PropsWithChildren<Histor
                                 (a, b) =>
                                     +new Date(b[1].lastInteractionTimestamp) - +new Date(a[1].lastInteractionTimestamp)
                             )
-                            .map(chat => {
-                                const lastMessage = chat[1].interactions[chat[1].interactions.length - 1].humanMessage
+                            .filter(
+                                ([, transcriptJSON]) =>
+                                    transcriptJSON.interactions && transcriptJSON.interactions.length > 0
+                            )
+                            .map(([id, transcriptJSON]) => {
+                                const lastMessage =
+                                    transcriptJSON.interactions[transcriptJSON.interactions.length - 1].humanMessage
                                 if (!lastMessage?.displayText) {
```

Comment:
```suggestion
                                if (!transcriptJSON.interactions.length || !lastMessage?.displayText) {
```
Is this the same as adding the filter?

Reply:
I don't think so? The error was caused by this line:
```
const lastMessage = chat[1].interactions[chat[1].interactions.length - 1].humanMessage
```

`chat[1].interactions` was an empty array, which means `chat[1].interactions[chat[1].interactions.length - 1]` was undefined.